### PR TITLE
[BUILD] Added Update script that checks for runtime package from Dawn

### PR DESCRIPTION
**Pull request by AVSurfer123**

### CODE REVIEW

File: scripts/update.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+done
+
+echo 'IMPORTANT: Runtime Rebooting for Update' >> /tmp/log-fifo
+sleep 2
+
+SERVICES="executor dev_handler net_handler shm_stop shm_start"
+
+cd systemd && sudo systemctl stop $SERVICES
+
+unzip -o /tmp/runtime.zip -d ~/
```
Comment (benliao1) [inline scripts/update.sh:24]: The only question I have here is like did you test to make sure this overwrites the existing runtime folder with no problems (does it ask for permission to overwrite the existing directory?) and on reboot it rebuilds Runtime? Like where/when is it rebuilding Runtime to get the updated executables?

**Review by benliao1 — COMMENTED** [review]
Just have one question about rebuilding runtime with this~~

File: scripts/update.sh (hunk 1 above)
Comment (AVSurfer123) [reply scripts/update.sh:24]: yes it overwrites the existing folder. it doesn't need to rebuild since the zip contains the built binaries

Comment (benliao1) [reply scripts/update.sh:24]: @AVSurfer123  wait how does the zip contain the built binaries? bc the .zip from the repo doesn't contain built binaries right? Unless Dawn scp's the .zip, unzips it, and then builds it?

Comment (benliao1) [reply scripts/update.sh:24]: sorry for the late reply

Comment (AVSurfer123) [reply scripts/update.sh:24]: oh, is the zip going to be the one directly from the github releases? I thought we could just host a zip of our working pi on the website for them to download. usually source code would be better, but every pi should be the exact same OS image here.

Comment (benliao1) [reply scripts/update.sh:24]: I'm not really sure, but I'm pretty sure we're not gonna post the entire raspi disk image on the website...

Comment (AVSurfer123) [reply scripts/update.sh:24]: I'm not saying to post the raspi disk image. I said its the ZIP of the runtime folder on a working raspi

**Review by benliao1 — APPROVED** [review]
Looks good! 👍

**Pull request merged**